### PR TITLE
Renamed WRange to WDist in the Lua API and documentation

### PR DESCRIPTION
**Pull request by Mailaender**
A followup of https://github.com/OpenRA/OpenRA/pull/8667.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Looks good to me. :+1:

File: OpenRA.Game/WDist.cs [hunk 1]
```diff
@@ -141,14 +141,14 @@ public LuaValue Equals(LuaRuntime runtime, LuaValue left, LuaValue right)
 			{
 				switch (key.ToString())
 				{
-					case "Range": return Length;
-					default: throw new LuaException("WPos does not define a member '{0}'".F(key));
+					case "Length": return Length;
```
Comment (pchote) [inline OpenRA.Game/WDist.cs]: This is going to break any script that calls .Range.  Please keep that as an alias, and have it log a message to the game chat saying it is deprecated.

File: OpenRA.Mods.Common/Scripting/Global/CoordinateGlobals.cs [hunk 2]
```diff
@@ -59,15 +59,15 @@ public class WVecGlobal : ScriptGlobal
 		public WVec Zero { get { return WVec.Zero; } }
 	}
 
-	[ScriptGlobal("WRange")]
-	public class WRangeGlobal : ScriptGlobal
+	[ScriptGlobal("WDist")]
```
Comment (pchote) [inline OpenRA.Mods.Common/Scripting/Global/CoordinateGlobals.cs]: This is going to break scripts.  Please keep both, and have WRange log deprecation notices to the game chat to encourage the map author to change it during the next release cycle.

**Comment by obrakmann** [comment]
Can you add this change on top, please?
```
(18:11:44) antares79: in #8667, WRange.Range was renamed to WDist.Length, but WRange.RangeSquared became WDist.RangeSquared. Should it be WDist.LengthSquared?
(18:11:57) pchote: yup
```

File: OpenRA.Game/WDist.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Game/WDist.cs]: Done.

**Comment by Mailaender** [comment]
Sure. Updated.

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/76c5c119651bd9023b67e47aed2117039930dc10%5E...76c5c119651bd9023b67e47aed2117039930dc10)

**Pull request merged**